src/java/org/apache/cassandra/io/sstable/format/SSTableWriter.java

```diff
@@ -437,6 +441,7 @@ public abstract static class Builder<W extends SSTableWriter, B extends Builder<
         private boolean transientSSTable;
         private SerializationHeader serializationHeader;
         private List<Index.Group> indexGroups;
+        private MutationIdRanges mutationIdRanges = MutationIdRanges.NONE;
```

Comment:
Can you require that `mutationId` ranges be provided? I don't think this is something that should have a default value. If it's going to be `NONE` make the caller provide `NONE`.